CHANGES.rst

```diff
@@ -8,6 +8,11 @@
 Changes
 =======
 
+Version 1.3.1 (released TBD)
+
+- set Sphinx ``<3`` because of errors related to application context
```

Comment:
minor: since we have to trigger build here (just to have one passing and not merge force, checked the passed link thanks for that ;)

Can it be changed to:
- Start items with capitals.
- End sentences with dots.
- Expand versions (i.e. 3.0.0).

Mainly to be consistent with the rest of the CHANGES.rst file (yes we need a common one for Invenio 😅 )